### ISSUE TITLE
DOTA test set split

### ISSUE THREAD

**Issue by MFalam** [ashnair1/dotadevkit#18]
Hello,
How to split the DOTA test set images that do not have labelTxt files?
Thank you

**Comment by ashnair1** [comment]
Refer the [split](https://github.com/ashnair1/dotadevkit#split) section in the README.

**Comment by MFalam** [comment]
Thanks, @ashnair1 for the prompt response.

I tried the following script and it splited the images.
`dotadevkit split \
        ./example/images \
        ./example_split/images \
        8 \
        --images`

However, I want to split the images into some particular rescale sizes and patches. I tried the following script.

` dotadevkit split \
        ./example/ \
        ./example_split/ \
        8 \
        800 \
        200 \
        0.0\
        0.5`

I got the following error:

_FileNotFoundError: [Errno 2] No such file or directory: 'example/images/images'_

I created another images directory inside the images directory, but it still looks for the .txt file of each image.
